cypress/e2e/signups.cy.js

```diff
@@ -2,7 +2,10 @@ import '@testing-library/cypress/add-commands'
 
 describe('Validate signup links', () => {
   beforeEach(() => {
-    cy.visit(Cypress.env('/signup'))
+    if (Cypress.env('isMobile')) {
+      cy.viewport(393,852)
+    }
+    cy.visit('/signup')
```

Comment:
I think we need add the base url, otherwise won't work
```suggestion
    cy.visit('/signup')
```